tox.ini

```diff
@@ -41,7 +41,7 @@ basepython = python3
 envdir={toxworkdir}/lint
 deps = -r{toxinidir}/requirements-lint.txt
 commands =
-  mypy {posargs}
+  mypy {posargs} .
```

Comment:
This might break when `tox -e mypy -- tests/unit/test_cli.py` is run for a single file/sub-dir check.